experimental/grouped-and-auto-accessors.md

```diff
@@ -0,0 +1,87 @@
+# [Grouped Accessors and Auto-Accessors][proposal-grouped-and-auto-accessors]
+
+## ClassBody
+
+```js
+extend interface ClassBody {
+    body: [ MethodDefinition | PropertyDefinition | StaticBlock | ClassAccessor ];
+}
+```
+
+## ObjectExpression
+
+```js
+extend interface ObjectExpression {
+    properties: [ Property | ObjectAccessor ];
+}
+```
+
+## ClassAccessor
+
+```js
+interface ClassAccessor <: PropertyDefinition {
+    type: "ClassAccessor";
+    get: GetAccessorStub | GetAccessorMethod | null;
+    set: SetAccessorStub | SetAccessorMethod | null;
+}
+```
+
+If `set.private` is `true`, `ClassAccessor#key` must be an `Identifier` and `ClassAccessor#computed` must be `false`.
+
+## ObjectAccessor
+
+```js
+interface ObjectAccessor <: Node {
+    type: "ObjectAccessor";
+    key: Literal | Identifier;
+    computed: boolean;
+    get: GetAccessorStub | GetAccessorMethod | null;
+    set: SetAccessorStub | SetAccessorMethod | null;
+}
```

Comment:
In the latest commit I removed the inheritance from `Property`, because it will inherit `value: Expression` and `kind: "init" | "set" | "get"`. These properties are not application to the grouped accessor.

Reply:
Makes sense.